packages/editor/src/components/post-author/index.js

```diff
@@ -11,7 +11,7 @@ import PostAuthorCombobox from './combobox';
 import PostAuthorSelect from './select';
 import { AUTHORS_QUERY } from './constants';
 
-const minimumUsersForCombobox = 25;
```

Comment:
This is only to make it easier to test for this fix. I will undo this change before merging, if this PR will get approved.